blueprints/ember-frost-bunsen/index.js

```diff
@@ -8,13 +8,13 @@ module.exports = {
         return this.addAddonsToProject({
           packages: [
             {name: 'ember-browserify', target: '^1.1.12'},
-            {name: 'ember-bunsen-core', target: '0.12.1'},
-            {name: 'ember-frost-core', target: '0.30.0'},
-            {name: 'ember-frost-fields', target: '^1.0.0'},
-            {name: 'ember-frost-tabs', target: '^3.1.0'},
+            {name: 'ember-bunsen-core', target: '~0.12.0'},
+            {name: 'ember-frost-core', target: '^1.0.0'},
+            {name: 'ember-frost-fields', target: '^4.0.0'},
+            {name: 'ember-frost-tabs', target: '^4.0.0'},
```

Comment:
There is a PR on `frost-tabs` that will bump it to `5.0.0`. I'm just waiting for the approval.